include/zephyr/net/wifi_mgmt.h

```diff
@@ -41,6 +41,12 @@ extern "C" {
 #define WIFI_MGMT_SCAN_SSID_FILT_MAX 1
 #endif /* CONFIG_WIFI_MGMT_SCAN_SSID_FILT_MAX */
 
+#ifdef CONFIG_WIFI_MGMT_SCAN_CHAN_MAX_MANUAL
```

Comment:
MANUAL doesn't add any value, so, we can drop that suffix.

Reply:
I was attempting to differentiate the name from `WIFI_CHANNEL_MAX`